src/components/manage/Sidebar/__snapshots__/SidebarPopup.test.jsx.snap

```diff
@@ -1,55 +1,59 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`sidebar popup is not rendered when the block is not selected 1`] = `
-<CSSTransition
-  classNames="sidebar-container"
-  in={false}
-  timeout={500}
-  unmountOnExit={true}
->
-  <Portal>
-    <aside
-      className="sidebar-container"
-      onClick={[Function]}
-      onKeyDown={[Function]}
-      role="presentation"
-      style={
-        Object {
-          "overflowY": "auto",
+<React.Fragment>
+  <CSSTransition
+    classNames="sidebar-container"
+    in={false}
+    timeout={500}
+    unmountOnExit={true}
+  >
+    <Portal>
+      <aside
+        className="sidebar-container"
+        onClick={[Function]}
+        onKeyDown={[Function]}
+        role="presentation"
+        style={
+          Object {
+            "overflowY": "auto",
+          }
         }
-      }
-    >
-      <p>
-        Tested, but you shouldn't see this in the snapshot!
-      </p>
-    </aside>
-  </Portal>
-</CSSTransition>
+      >
+        <p>
+          Tested, but you shouldn't see this in the snapshot!
```

Comment:
😅

Reply:
Looks like it used to be in the old snapshot as well.